python/gui/MaaS/routing.py

```diff
@@ -0,0 +1,13 @@
+"""
+Defines the routes that will be acknowledged within the app to connect a websocket url to its consumer
+"""
+import typing
+
+from django.urls import re_path
+from django.urls.resolvers import URLResolver
+from django.urls.resolvers import URLPattern
+
+
+websocket_urlpatterns = [
+    # re_path(f'ws/example/route', consumers.ExampleConsumer.as_asgi(), name="ExampleSocketConsumer")
```

Comment:
Assuming this is purposefully commented out, yeah?

Reply:
Yeah, it's just there for an example.